components/elm/src/biogeophys/CanopyFluxesMod.F90

```diff
@@ -47,6 +47,7 @@ module CanopyFluxesMod
   use ColumnDataType        , only : col_es, col_ef, col_ws               
   use VegetationType        , only : veg_pp                
   use VegetationDataType    , only : veg_es, veg_ef, veg_ws, veg_wf  
+  use spmdmod          , only: masterproc
```

Comment:
Looks like this insertion of masterproc is not needed

Reply:
@thorntonpe I have removed insertion in my latest commit. Thanks.